plugins/catalog-graph/src/components/CatalogGraphPage/SelectedOwnbyFilter.tsx

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
This filename has the wrong casing it seems. It should be named `SelectedOwnedByFilter` exactly